packages/coil-extension/src/background/services/BackgroundScript.ts

```diff
@@ -321,18 +329,29 @@ export class BackgroundScript {
         const frameStates = Object.values(tabState.frameStates)
         const hasStream = frameStates.find(f => f.monetized)
         const hasBeenPaid = hasStream && frameStates.find(f => f.total > 0)
+        const paused = tabState.playState === 'paused'
+        const disabled = Object.values(tabState.disabling).some(Boolean)
 
         if (hasStream) {
+          // this.tabStates.setIcon(tabId, 'inactive')
           this.tabStates.setIcon(tabId, 'monetized')
+
           if (hasBeenPaid) {
-            const state =
-              tabState.playState === 'playing'
-                ? 'streaming'
-                : 'streaming-paused'
-            this.tabStates.setIcon(tabId, state)
+            this.tabStates.setIcon(tabId, 'streaming')
```

Comment:
Note that the state of the icon is actually a combination of these setIcon calls. It's probably not very obvious to those uninitiated

Reply:
````